src/index.js

```diff
@@ -32,12 +34,26 @@ const App = () => {
           actions.changeQuantity,
           actions.emptyCart,
           actions.readdProduct,
-          actions.readdOrder
+          actions.readdOrder,
+
+          actions.setCustomerId,
+          actions.removeCustomerId
         ],
         (state, _) => {
           state.totalPrice = state.products.reduce((acc, cur) => acc + cur.price * cur.quantity, 0);
         }
       ),
+      
+      // -- CUSTOMER ACTIONS --
+
+      // Set the current customerId
+      setCustomerId: action((state, customerId) => {
+        state.customerId = customerId;
+      }),
+
+      removeCustomerId: action((state) =>{
+        state.customerId = null;
+      }),
```

Comment:
Move the customer actions out of the cart state into customer state

Reply:
moved